ds-caselaw-ingester/lambda_function.py

```diff
@@ -505,12 +515,23 @@ def process_message(message):
     if has_TDR_data:
         store_metadata(uri, metadata)
 
+    # Determine if there's a word document -- we need to know before we save the tar.gz file
+    docx_filename = extract_docx_filename(metadata, consignment_reference)
+    print(f"extracted docx filename is {docx_filename!r}")
+
     # Copy original tarfile
-    store_file(open(filename, mode="rb"), uri, os.path.basename(filename), s3_client)
+    modified_targz_filename = (
```

Comment:
Verry nitpicky and non-blocking, but since it's not always modified, would it preferable to call it `targz_filename`?

Reply:
I'm going to leave this one, since the tar_gz file already has a filename, and it's the one that (might have been) modified.